tools/kagami/Cargo.toml

```diff
@@ -16,10 +16,9 @@ iroha_data_model = { version = "=2.0.0-pre-rc.13", path = "../../data_model" }
 iroha_schema_gen = { version = "=2.0.0-pre-rc.13", path = "../../schema/gen" }
 iroha_schema = { version = "=2.0.0-pre-rc.13", path = "../../schema" }
 iroha_primitives = { version = "2.0.0-pre-rc.13", path = "../../primitives" }
-iroha_wasm_builder = { version = "=2.0.0-pre-rc.13", path = "../../wasm_builder" }
 iroha_genesis = { version = "=2.0.0-pre-rc.13", path = "../../genesis" }
+iroha  = { version = "=2.0.0-pre-rc.13", path = "../../cli" }
```

Comment:
I'm also not very happy about this

Reply:
I would rather you duplicate the `construct_validator`

Reply:
Yes, I used it for `construct_validator()`. I thought it will be good to place our validator into `cli/samples` because it's a sample in fact. And our `kagami` also generates sample configs. So what's the problem?

Reply:
the concern is compile times. Dependency graph of `kagami` will now include `iroha_core` but for no reason.
We've gone through the trouble of disecting the project into smaller crates so we don't have one big monolithic app

Reply:
Okay, if so, I'll copy-paste this function